vpr/src/server/telegrambuffer.h

```diff
@@ -70,7 +75,7 @@ class TelegramBuffer
      *
      * @note After calling this function, the internal error storage will be cleared.
      */
-    void take_errors(std::vector<std::string>&);
+    void take_errors(std::vector<std::string>& errors);
 
     /**
      * @brief Retrieves a constant reference to the internal byte buffer.
```

Comment:
I'd also comment check_telegram_header_presence().
Also good to comment what the private member data is too.

Reply:
solved c445d93a14b06bfc1feedfe9fc949803a26cca8a